### PR TITLE
feat(deploy.py): support w pack qkv

### PR DESCRIPTION
**Pull request by tpoisonooo**
#79 

Add Baichuan-7B-Chat support.  https://huggingface.co/csdc-atl/baichuan-7B-chat

### CODE REVIEW

File: lmdeploy/serve/turbomind/deploy.py [hunk 1]
```diff
@@ -323,14 +323,32 @@ def get_tensor_transposed(name: str):
...
-                                 map(('{}.' + suffix).format, _qkvo))
+                if w_pack:
+                    _qkvo = [f'model.layers.{i}.self_attn.{t}' for t in ['W_pack', 'o_proj']]
+                    qkv, o = map(get_tensor_transposed,
+                                    map(('{}.' + suffix).format, _qkvo))
+
+                    if qkv is None:
+                        continue
+                    _shape = qkv.shape[0]
+                    qkv = torch.split(qkv, [_shape, _shape, _shape], dim=1)
```
Comment (AllentDan) [inline lmdeploy/serve/turbomind/deploy.py:343]: So the qkv.shape[1] always equals 3*qkv.shape[0]?

**Review by AllentDan — APPROVED** [review]
LGTM

File: lmdeploy/serve/turbomind/deploy.py (hunk 1 above)
Comment (tpoisonooo) [reply lmdeploy/serve/turbomind/deploy.py:343]: Yes, W_pack means packing qkv

Comment (tpoisonooo) [reply lmdeploy/serve/turbomind/deploy.py:343]: fixed.

File: README.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ python3 lmdeploy.app {server_ip_addresss}:33337 internlm
 
 ![](https://github.com/InternLM/lmdeploy/assets/67539920/08d1e6f2-3767-44d5-8654-c85767cec2ab)
 
-For the deployment of other supported models, such as LLaMA, vicuna, you can find the guide from [here](docs/en/serving.md)
+For the deployment of other supported models, such as LLaMA, vicuna and Baicuan, you can find the guide from [here](docs/en/serving.md)
```
Comment (grimoire) [inline README.md:123]: typo

**Comment by grimoire** [comment]
Register new model in `lmdeploy/model.py` for initialize prompt and stop words.

File: README.md (hunk 2 above)
Comment (tpoisonooo) [reply README.md]: done.

**Review by grimoire — APPROVED** [review]
LGTM

**Pull request merged**